### PR TITLE
[Storage][Blob][STG71]CPK-N

### PR DESCRIPTION
**Pull request by xiafu-msft**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/storage/azure-storage-blob/azure/storage/blob/_models.py [hunk 1]
```diff
@@ -1014,6 +1018,42 @@ def from_string(cls, permission):
         return parsed
 
 
+class ContainerCpkScopeInfo(ContainerCpkScopeInfoGen):
+    """This scope is then used implicitly for all future writes within the container,
+    but can be overridden per-request via explicit request headers.
+
+    :param default_encryption_scope: Optional.  Version 2019-02-02 and later.
```
Comment (zezha-msft) [inline sdk/storage/azure-storage-blob/azure/storage/blob/_models.py]: > 2019-02-02 [](start = 55, length = 11)

shouldn't the version be 2019-07-07 instead?

Comment (zezha-msft) [inline sdk/storage/azure-storage-blob/azure/storage/blob/_models.py]: Minor: extra spaces

File: sdk/storage/azure-storage-blob/tests/test_cpk_n.py [hunk 2]
```diff
@@ -0,0 +1,699 @@
...
+    BlobSasPermissions,
+    CpkScopeInfo,
+    ContainerCpkScopeInfo,
+    generate_blob_sas
+)
+from _shared.testcase import StorageTestCase, GlobalStorageAccountPreparer
+
+# ------------------------------------------------------------------------------
+
+TEST_ENCRYPTION_KEY_SCOPE = CpkScopeInfo(encryption_scope="antjoscope1")
```
Comment (zezha-msft) [inline sdk/storage/azure-storage-blob/tests/test_cpk_n.py]: >CpkScopeInfo(encryption_scope="antjoscope1") [](start = 28, length = 44)

Please leave a comment to explain why these values are hard-coded.

File: sdk/storage/azure-storage-blob/tests/test_cpk_n.py [hunk 3]
```diff
@@ -0,0 +1,699 @@
...
+        return blob
+
+    # -- Test cases for APIs supporting CPK ----------------------------------------------
+
+    @pytest.mark.playback_test_only
+    @GlobalStorageAccountPreparer()
+    def test_put_block_and_put_block_list(self, resource_group, location, storage_account, storage_account_key):
+        # Arrange
+        # test chunking functionality by reducing the size of each chunk,
+        # otherwise the tests would take too long to execute
```
Comment (zezha-msft) [inline sdk/storage/azure-storage-blob/tests/test_cpk_n.py]: there's no chunking in this test though?

File: sdk/storage/azure-storage-blob/azure/storage/blob/_models.py (hunk 1 above)
Comment (xiafu-msft) [reply sdk/storage/azure-storage-blob/azure/storage/blob/_models.py]: Good catch

Comment (xiafu-msft) [reply sdk/storage/azure-storage-blob/azure/storage/blob/_models.py]: Yeah, good catch. I forgot to check when copy paste from generated code

**Pull request merged**